cmd/kube-apiserver/app/server.go

```diff
@@ -51,7 +51,8 @@ import (
 const (
 	// Maximum duration before timing out read/write requests
 	// Set to a value larger than the timeouts in each watch server.
-	ReadWriteTimeout = time.Minute * 60
+	ReadWriteTimeout            = time.Minute * 60
+	defaultLongRunningRequestRE = "(/watch(/.*|)$|/proxy(/.*|)$|logs(/|)$|portforward(/|)$|exec(/|)$)"
```

Comment:
It can be somewhat simplified. It's my understanding that /watch/ and /proxy/ can have more path after but portforward, logs, and exec have to be at the end of the path. Is that incorrect?